annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/Serializer.java

```diff
@@ -61,19 +61,21 @@ public class Serializer {
   public Serializer(Config config) {
     Path outputDirectory = config.getOutputDirectory();
     this.fieldGraphPath = outputDirectory.resolve(FIELD_GRAPH_FILE_NAME);
-    this.callGraphPath = outputDirectory.resolve(CALL_GRAPH_FILE_NAME);
+    this.methodImpactedRegion = outputDirectory.resolve(METHOD_IMPACTED_REGION_FILE_NAME);
     this.methodInfoPath = outputDirectory.resolve(METHOD_INFO_FILE_NAME);
     this.classInfoPath = outputDirectory.resolve(CLASS_INFO_FILE_NAME);
     initializeOutputFiles(config);
   }
 
   /**
-   * Appends the string representation of the {@link TrackerNode} corresponding to a call graph.
+   * Appends the string representation of the {@link TrackerNode} corresponding to method impacted
+   * region.
    *
-   * @param callGraphNode TrackerNode instance.
+   * @param methodImpactedRegion TrackerNode instance which is an impacted region for a method
+   *     change.
    */
-  public void serializeCallGraphNode(TrackerNode callGraphNode) {
-    appendToFile(callGraphNode.toString(), this.callGraphPath);
+  public void serializeImpactedRegionByMethod(TrackerNode methodImpactedRegion) {
```

Comment:
What do you think of this?
```suggestion
  public void serializeImpactedMethodRegion(TrackerNode impactedMethodRegion) {
```
To me this is clearer.

Reply:
@msridhar In this method, we only serialize a region that can be impacted for a change on a __method__ (not a change on a field) and the serialized region can be a method or a field. Do you still think this is clearer considering its purpose. I think `serializeImpactedMethodRegion` might sound as a method for serializing method regions that can be impacted either by change on method or field. Either way, I will update javadoc to describe this clearer once the name is finalized.

Reply:
Ah ok.  I misunderstood.  Then how about `serializeImpactedRegionForMethod`, and also add corresponding javadoc to explain?

Reply:
d6a6944f4dbdbb746fe6f639e8103cad21d9dfbc